MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -207,9 +215,11 @@ private static IEnumerable<Directory> ProcessChunk([NotNull] PngChunk chunk)
             else if (chunkType == PngChunkType.tEXt)
             {
                 var reader = new SequentialByteArrayReader(bytes);
-                var keyword = reader.GetNullTerminatedString(maxLengthBytes: 79);
+                var keyword = reader.GetNullTerminatedStringValue(maxLengthBytes: 79).ToString(defaultEncoding);
```

Comment:
I went back and forth on this one originally, and agree - it's a toss-up. You could certainly pass it along, but in the end it seemed like the reader should stick to 'reading' and let the user decide later how it should be interpreted explicitly. An overload could accomplish it in the near term; if it was required up-front and the default encoding for that reader always matched, it would seem a little redundant. But again, this feels like a programmer 'taste' issue.